recipes/cubicinterpolation/all/conandata.yml

```diff
@@ -1,17 +1,21 @@
 sources:
   "0.1.5":
-    url: "https://github.com/MaxSac/cubic_interpolation/archive/v0.1.5.tar.gz"
+    url: "https://github.com/tudo-astroparticlephysics/cubic_interpolation/archive/v0.1.5.tar.gz"
     sha256: "fc34de15c9dd9e651728c9e0eee5528ee9636e41a3d8aa6f41735018810afd59"
   "0.1.4":
-    url: "https://github.com/MaxSac/cubic_interpolation/archive/v0.1.4.tar.gz"
+    url: "https://github.com/tudo-astroparticlephysics/cubic_interpolation/archive/v0.1.4.tar.gz"
     sha256: "38bb8fe46b19b135bfcba79e098c5d90284f0bc02f42f86118aefcb63aed7668"
   "0.1.3":
-    url: "https://github.com/MaxSac/cubic_interpolation/archive/v0.1.3.tar.gz"
+    url: "https://github.com/tudo-astroparticlephysics/cubic_interpolation/archive/v0.1.3.tar.gz"
     sha256: "3151d99ecbbddd4e57605ff1919bdf234d08336b47d369b9dc562acff780aaf7"
 patches:
   "0.1.4":
     - patch_file: "patches/patch_conanbuildinfo.txt"
-      base_path: "source_subfolder"
+      patch_type: "conan"
+    - patch_file: "patches/patch_conan_basic_setup.txt"
```

Comment:
Would you mind renaming this (and  the old patch) to a `.diff` this way github will apply formating and it will be easier to read 🙏 

`patches/rm_conan_basic_setup.diff` would be amazing 🤩